client/landing/jetpack-cloud/sections/backups/rewind-flow/index.tsx

```diff
@@ -28,20 +28,26 @@ import { applySiteOffset } from 'lib/site/timezone';
 import './style.scss';
 
 interface Props {
-	rewindId?: string;
+	rewindId: string;
 	purpose: RewindFlowPurpose;
 }
 
 const BackupRewindFlow: FunctionComponent< Props > = ( { rewindId, purpose } ) => {
 	const translate = useTranslate();
 	const siteId = useSelector( getSelectedSiteId );
 
-	const timezone = useSelector( state => getSiteTimezoneValue( state, siteId ) );
-	const gmtOffset = useSelector( state => getSiteGmtOffset( state, siteId ) );
+	const timezone = useSelector( state =>
+		siteId !== null ? getSiteTimezoneValue( state, siteId ) : null
```

Comment:
Could this happen? `siteId === null` If we are not in a site, we should reject the request to this page.

Reply:
this construct prevents the typescript from complaining